### ISSUE TITLE
Chrono 0.3.1 depends on serde 1.0.1 which breaks with diesel 0.12 dependencies

### ISSUE THREAD

**Issue by Nukesor** [diesel-rs/diesel#884]
If you declare a Module with `#[derive(Debug, Serialize, Deserialize, Queryable)]` and this model contains a `chrono::NaiveDateTime;` the compiler warns about a missing trait.

```
7 | #[derive(Debug, Serialize, Deserialize, Queryable)]
  |                 ^^^^^^^^^ the trait `validation::pod::_IMPL_DESERIALIZE_FOR_PodSettingsSerializer::_serde::Serialize` is not implemented for `chrono::NaiveDateTime`
```

This seems to be a conflict between serde versions for chrono.

**Comment by Eijebong** [comment]
https://github.com/chronotope/chrono/commit/dcf193323b59b8c85f0be708d5ba2b6a1a1654d2

:(

**Comment by Nukesor** [comment]
Ok. `cargo update -p chrono --precise 0.3.0` temporarily fixes this issue.

**Comment by sgrif** [comment]
I've published 0.12.1 which locks the chrono version
